docs/src/pages/components/cdk-components/text-input/examples/lengthTextInput.js

```diff
@@ -3,7 +3,7 @@ import { useState } from "react";
 
 const code = `() => {
   const [value, setValue] = useState("");
-  const [error, setError] = useState("");
```

Comment:
In this particular example, you don't need to remove the initial value `""` of the `error` variable, since we are using the predefined error message. It also adds some value, since if they are using typescript it will infer the type of the stateful variable (`string` in this case, not `undefined` as it currently suggests).